packages/teams-js/src/public/location.ts

```diff
@@ -12,6 +12,11 @@ import { runtime } from './runtime';
  * Namespace to interact with the location module-specific part of the SDK.
  */
 export namespace location {
+  /** Get location callback function type */
+  export type getLocationCallbackFunctionType = (error: SdkError, location: Location) => void;
```

Comment:
Same comment about not introducing new exported types in this change

Reply:
In latest commit didn't expose this function.